csrc/ops/arith.h

```diff
@@ -736,21 +742,36 @@ NVF_API TensorView* argsort(
 //! Grouped matrix multiplication
 //!
 //! Performs matrix multiplication on grouped sets of matrices using offsets
-//! to define variable-sized groups.
+//! to define variable-sized groups. This op computes:
+//!
+//! alpha * grouped_mm((mat1 * scale1), (mat2 * scale2), offsets) + bias * beta
 //!
 //! \param mat1 First set of matrices
 //! \param mat2 Second set of matrices
 //! \param offsets Offsets tensor defining group boundaries
 //! \param scale1 Scale tensor for mat1
 //! \param scale2 Scale tensor for mat2
+//! \param alpha Global Scaling factor for mat1@mat2
+//! \param bias Bias tensor
+//! \param beta Scale tensor for bias
+//! \param dtype Output dtype
+//! \param out_block_scale_size Output block scaling factor size
+//! \param block_scaling_factor_dtype Block scaling factor dtype
+//! \param out_gamma Output gamma flag
 //! \return Result of grouped matrix multiplication
-NVF_API TensorView* grouped_mm(
+NVF_API ScaledTensorView grouped_mm(
```

Comment:
```suggestion
ScaledTensorView grouped_mm(
```

Reply:
This is part of the c++ API and we do want to expose this.

Any reason we suggest to hide it?